build/profiles/freenas/packages/freenasUI/config

```diff
@@ -65,6 +65,9 @@ post-upgrade =
              fi
 	     rm -f /tmp/.sqlite3_ha_skip
 
+             # vcp got removed and this is to ensure that the relevant directory is deleted if upgrader fails to remove it
+             rm -rf /usr/local/www/freenasUI/vcp
```

Comment:
This has to happen before migration runs.
You also seem to be using spaces instead of tabs